app/components/Card/CardBack.tsx

```diff
@@ -16,6 +16,7 @@ interface CardBackProps {
   setIsFlipped: React.Dispatch<React.SetStateAction<boolean>>;
   name?: string | null;
   description?: TinaMarkdownContent | TinaMarkdownContent[];
+  path: string;
 }
 
 const CardBack: React.FC<CardBackProps> = ({
```

Comment:
do you think we should have syntax consistency in how we set our React component function declarations? when there are several props this syntax you have here makes much more sense vs this:
```
export default function CardBack({
  animation,
  isFlipped,
  setIsFlipped,
  name,
  description,
  path,
}: CardBackProps) {
```
but then we're declaring functions two separate ways. Do you think it's best practice to keep it one way?

Reply:
Good point! I agree that it would be great if we keep syntax consistent and declare functions in one way. I don't necessarily think that the way I set up this component makes more sense than the snippet you provided, as I destructure props either way, so I'll just incorporate your suggestion :)